package.json

```diff
@@ -1,18 +1,26 @@
 {
-  "name": "Alinka",
+  "name": "alinka",
```

Comment:
Just asking. Is lowercase a common approach for naming package?

Reply:
yup
https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields